### ISSUE TITLE
QUESTION: How to avoid collapsing during change item in accordion

### ISSUE THREAD

**Issue by zion03** [kaliber5/ember-bootstrap#563]
I want open all accordion items and collapsed / open item on click. 
This look like unsupported feature. What's the right way to do it?

**Comment by simonihmig** [comment]
No, the accordion component does not work like this. But it used `{{bs-collapse}}` internally, and you could use it as well to implement what you want. Something along the lines of this: (not tested, using BS3 markup)

```hbs
<h4 class="panel-title"><a href="#" {{action "toggle" item1}}>Item 1</a></h4>
{{#bs-collapse ariaRole="tabpanel" collapsed=item1 class="panel-collapse"}}
  <div class="panel-body">
    content 1
  </div>
{{/bs-collapse}}
```

Repeat that for every item. The toggle action would then call `toggleProperty()` for the given property.

Hope that helps!?
